src/main/java/Event.java

```diff
@@ -0,0 +1,15 @@
+public class Event extends Task{
+    protected String startTime;
+    protected String endTime;
+
+    public Event(String taskDescription, String startTime, String endTime) {
+        super(taskDescription);
+        this.startTime = startTime;
+        this.endTime = endTime;
+    }
+
+    @Override
+    public String toString(){
+        return "[E][" + getStatusIcon() + "] " + taskDescription + " (from: " + startTime  + " to: " + endTime + ")" ;
```

Comment:
You may consider using line wrapping to keep line length short.
```suggestion
        return "[E][" + getStatusIcon() + "] " + taskDescription 
                + " (from: " + startTime  + " to: " + endTime + ")" ;
```